desktop/core/src/desktop/js/apps/tableBrowser/metastoreViewModel.js

```diff
@@ -351,11 +354,16 @@ class MetastoreViewModel {
   }
 
   loadUrl() {
-    const path = window.location.pathname.substr(4) || '/metastore/tables';
+    const path = window.location.pathname.startsWith(window.HUE_BASE_URL)
+      ? window.location.pathname.substr(window.HUE_BASE_URL.length)
+      : window.location.pathname.substr(4) || '/metastore/tables';
     const pathParts = path.split('/');
     if (pathParts[0] === '') {
       pathParts.shift();
     }
+    while (pathParts[0] === 'hue') {
```

Comment:
while bit fishy to understand why, maybe adding a comment with example?

Reply:
@romainr 
When Knox is enabled, the Hue URL is /gateway/cdp-proxy/hue/hue/metastore/table/default/web_logs. Even though I don't see 2 'hue' in path happened inside metastoreViewModel.js, just want to be a bit more protective. And it will always have a 'metastore', so this should be safe. 

What do you think?

Reply:
For me it looked simpler to handle '/hue/hue/' strip on line 358, as there was already a half a change there.

But not a big issue